README.md

```diff
@@ -177,4 +178,5 @@ This bundle is released under the [MIT license](src/Resources/meta/LICENSE)
 [4]: http://docs.guzzlephp.org/en/latest/request-options.html
 [5]: https://github.com/gregurco/GuzzleBundleWssePlugin
 [6]: https://github.com/gregurco/GuzzleBundleCachePlugin
-[7]: https://github.com/8p/GuzzleBundle/graphs/contributors
+[7]: https://github.com/gregurco/GuzzleBundleOAuth2Plugin
+[8]: https://github.com/8p/GuzzleBundle/graphs/contributors
```

Comment:
Can you fix this link also? I know, it's not a modification by you...

Reply:
yep, no problem :slightly_smiling_face: